cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,49 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'userTest1',
+    password: 'Qwerty!23'
+  };
 
+  const book = {
+    title: 'Speaking JavaScript',
+    description: 'Like it or not, JavaScript is everywhere'
+  };
+  beforeEach(() => {
+    cy.visit('/login');
   });
 
-  it('', () => {
+  it('User is able to login', () => {
+    cy.findByPlaceholder('UserName').type(user.username);
+    cy.findByPlaceholder('Password').type(user.password);
+    cy.get('#login').click();
+    cy.get('#userName-value').should('contain', user.username);
+    cy.url().should('include', '/profile');
+  });
 
+  it('User is able to add a book', () => {
+    cy.login();
+    cy.visit('/profile');
+    cy.contains('#item-2', 'Book Store').click();
+    cy.findByPlaceholder('Type to search').type(book.title);
+    cy.contains('a', book.title).click();
+    cy.get('#description-wrapper').should('contain', book.description);
+    cy.get('.text-right > #addNewRecordButton').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`Book added to your collection.`);
+    });
+    cy.visit('/profile');
+    cy.contains('a', book.title).should('be.visible');
+  });
+  it('User is able to delete a book', () => {
```

Comment:
if you will run only this test - it'll fail, because there isn't added book for deletion. Autotests should be independent